### PR TITLE
LocalCheckout

### PR DESCRIPTION
**Pull request by nleach999**
Feature use is described in the README.  It should be backwards compatible with existing scripts, so changes are transparent until intentionally invoked.

### CODE REVIEW

**Comment by miguelfreitas93** [comment]
@nleach999 can you please resolve the conflicts before merging ?

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ latest (9.0) : https://download.checkmarx.com/9.0.0/Plugins/CxConsolePlugin-2020
 2020.2.7 : https://download.checkmarx.com/9.0.0/Plugins/CxConsolePlugin-2020.2.7.zip
 
 2020.2.3 : https://download.checkmarx.com/9.0.0/Plugins/CxConsolePlugin-2020.2.3.zip
+<<<<<<< HEAD
```
Comment (miguelfreitas93) [inline README.md]: @nleach999 seems like the merge didn't went ok

Comment (nleach999) [reply README.md]: ok, I think I got it right this time

File: import_certs.sh [hunk 2]
```diff
@@ -4,5 +4,5 @@ cd `dirname $0`
 
 for certfile in $(ls *.crt *.cer 2> /dev/null)
 do
-    keytool -importcert -file $certfile -alias $certfile -keystore $JAVA_HOME/jre/lib/security/cacerts -storepass changeit -noprompt
```
Comment (miguelfreitas93) [inline import_certs.sh:7]: @nleach999 why Keystore flag was removed ?

File: README.md [hunk 3]
```diff
@@ -120,6 +120,29 @@ Run Container:
 docker run my_custom_cxcli:latest Scan -CxServer http://localhost -CxUser admin@cx -CxPassword password -ProjectName /CxServer/SP/Company/Team/myproject -LocationType folder -LocationPath ./mycode -Log log.log -v
 ```
 
-### License
```
Comment (miguelfreitas93) [inline README.md:123]: @nleach999 why License details were removed from Readme ?

File: .github/workflows/container_scanning.yml [hunk 4]
```diff
@@ -19,7 +19,7 @@ jobs:
     - uses: anchore/scan-action@master
       with:
         dockerfile-path: "Dockerfile"
-        image-reference: "openjdk:8-jdk-alpine"
+        image-reference: "openjdk:oraclelinux7"
```
Comment (miguelfreitas93) [inline .github/workflows/container_scanning.yml:22]: @nleach999 why the image was changed ?

File: README.md (hunk 3 above)
Comment (nleach999) [reply README.md:123]: The license is in the root of the repo.

File: .github/workflows/container_scanning.yml (hunk 4 above)
Comment (nleach999) [reply .github/workflows/container_scanning.yml:22]: Alpine Linux lacks support for packages such as the Perforce client.  Most enterprise tools (assuming someone wants to use some other SCM or build tool in a child image) will likely want a RedHat derivative.  Oracle Linux is a fairly common choice for enterprise Linux.

Comment (miguelfreitas93) [reply .github/workflows/container_scanning.yml:22]: @nleach999 good to know, thanks ;)

File: import_certs.sh (hunk 2 above)
Comment (nleach999) [reply import_certs.sh:7]: -cacerts does the same thing without having to resolve the path to the cacerts keystore.

Comment (miguelfreitas93) [reply import_certs.sh:7]: oh, good to know, didn't knew about that option, thanks !

**Pull request merged**